### PR TITLE
Marathon Nexus Publish on Release

### PR DESCRIPTION
**Pull request by kensipe**
Marathon Nexus Publish on Release

Summary:
BP of Marathon Nexus Publish on Release and Release 1.5 does NOT get tested against DCOS only master.

JIRA issues:  MARATHON_EE-1752

### CODE REVIEW

**Comment by timcharper** [comment]
This is a backport but seems to change quite a bit more than https://github.com/mesosphere/marathon/pull/5775/files does; can you explain?

File: ci/pipeline [hunk 1]
```diff
@@ -201,12 +201,7 @@ def master(): Unit = {
   buildAndTestDockerAndLinuxPackages()
 
   // Uploads
-  val maybeArtifact = uploadTarballPackagesToS3(version, "builds")
-  maybeArtifact.foreach { artifact =>
```
Comment (alenkacz) [inline ci/pipeline]: this is something that was not removed on the master branch here https://github.com/mesosphere/marathon/pull/5775/files

was it intentional to remove this?

File: ci/pipeline [hunk 2]
```diff
@@ -279,11 +274,12 @@ def release(requestVersion: String, gitSha: String, latest: Boolean = false): Un
     println("Docker image mesosphere/marathon:latest NOT updated")
   }
 
+  %('sbt, "publish")
```
Comment (alenkacz) [inline ci/pipeline]: as @timcharper said in the other review, it would be nice to put a comment here...

**Comment by kensipe** [comment]
PR satisfies https://jira.mesosphere.com/browse/MARATHON_EE-1752

**Comment by kensipe** [comment]
@alenkacz @timcharper   The only difference in this backport from the original is the removal of code that runs automated integration to DCOS which is intentional and is in the comments of this PR.

**Comment by kensipe** [comment]
@mesosphere-ci please retest this

**Comment by kensipe** [comment]
@jeschkies @zen-dog @timcharper These files have nothing to do with the CI testing failures.    There is a CI reported green followed by a failure.   We really need to get a handle on these false failures.   These are TestFailedDueToTimeoutException failures.   I'm merging to move the ball forward.

**Pull request merged**